### PR TITLE
Funtionality for checking duplicate number when selected from contacts

### PR DESCRIPTION
**Pull request by AkankshaBodhankar**
Fixed issue #137

### CODE REVIEW

**Comment by AkankshaBodhankar** [comment]
After picking the same contact again
Duplicate number message toast gets displayed

screenshot_2016-03-13-17-50-53_com peacecorps pcsa

File: app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java [hunk 1]
```diff
@@ -237,6 +244,23 @@ private void showNoPhoneNumberToast() {
         Toast.makeText(Trustees.this, R.string.no_phone_number, Toast.LENGTH_LONG).show();
     }
     
+    private boolean check_duplicate_contact_number(String selectedNumber)
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java]: This is not the correct convention of naming methods in Java.

**Comment by sandarumk** [comment]
Since this is also closely related to your next PR, Can you please do the modification as requested and submit a separate PR solving both the issues?

**Comment by AkankshaBodhankar** [comment]
@sandarumk Sure will make a PR soon

**Comment by AkankshaBodhankar** [comment]
closing the issue and making a seperate PR

**Pull request closed without merging**